src/category_theory/instances/rel.lean

```diff
@@ -0,0 +1,12 @@
+import category_theory.category
+
+namespace category_theory
+
+universe u
+
+def rel : large_category (Type u) :=
```

Comment:
```suggestion
def rel : large_category (Sort u) :=
```
Shouldn't this be `Sort` as well? Or is this one of the problematic cases?

Reply:
This is one of the problematic cases. I've added a comment at this definition.